src/FloorSubsystem.java

```diff
@@ -0,0 +1,90 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.text.ParseException;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+/**
+ * The FloorSubsystem Class parses events and processes the information from the
+ * events, sending floorEvents and taking arrival events from scheduler
+ * 
+ */
+public class FloorSubsystem implements Runnable {
+
+	private Queue<FloorEvent> eventList;
+	private List<Floor> floors;
+	private int numOfFloors;
+	private MiddleMan box;
```

Comment:
i think the scheduler and elevator subsystem call the MiddleMan object 'middleMan', might be good to keep that consistent